app/notification/application/map_contents.py

```diff
@@ -34,15 +35,16 @@ def from_json(json_data: dict):
             ApplicationData object with application contents.
         """
         data = Notification.from_json(json_data)
-        application = data.content["application"]
-        return Application(
-            contact_info=data.contact_info,
-            questions=Application.process_questions_and_answers(data),
-            submission_date=application.get("date_submitted"),
-            fund_name=application.get("project_name"),
-            fund_round=application.get("round_id"),
-            application_id=application.get("id"),
-        )
+        if data.template_type == "APPLICATION_RECORD_OF_SUBMISSION":
+            application = data.content["application"]
+            return Application(
+                contact_info=data.contact_info,
+                questions=Application.bytes_object_for_questions_answers(data),
+                submission_date=application.get("date_submitted"),
```

Comment:
I thought we were using the constants here?  https://github.com/communitiesuk/funding-service-design-notification/blob/eedd7ef7f0312c928f38491bc7339a9bf0e9c122/app/notification/magic_link/map_contents.py#L31

Reply:
I pushed some changes instead of stashing. The PR was mainly about application-conformation email with file attachment but I was doing some other study while waiting for your merge so please  ignore it. I will be refactoring them as discussed